modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -0,0 +1,164 @@
+import * as t from 'io-ts';
+
+export interface CreateInvoiceParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface CreateDepositAddressParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
```

Comment:
I don't think the deposit address will ever expire so we can drop this.

Reply:
We don't have a `memo` either.

Reply:
it also doesn't have an amount